packages/commons-test/package.json

```diff
@@ -33,6 +34,7 @@
     "typescript": "5.1.3",
     "uuid": "9.0.1",
     "vitest": "0.33.0",
```

Comment:
Shouldn't we also use the latest version here in commons-tests to make the new setup & types work?